wallet/wallet.go

```diff
@@ -4,70 +4,358 @@ package wallet
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/multisig"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/utils"
+
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
 
-	"github.com/ava-labs/avalanche-tooling-sdk-go/keychain"
-	"github.com/ava-labs/avalanchego/ids"
-	avagokeychain "github.com/ava-labs/avalanchego/utils/crypto/keychain"
 	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
-	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
 	"github.com/ava-labs/avalanchego/wallet/subnet/primary/common"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/tx"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/account"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet/p-chain/txs"
+	"github.com/ava-labs/avalanchego/ids"
+	avagoTxs "github.com/ava-labs/avalanchego/vms/platformvm/txs"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+	"github.com/ava-labs/subnet-evm/ethclient"
 )
 
-type Wallet struct {
+type ChainClients struct {
```

Comment:
How are we going to be using clients?

Reply:
When we call info apis like get subnet owners, etc